build/main.mk

```diff
@@ -804,7 +804,7 @@ XCSOAR_DEPENDS = GETTEXT PROFILE \
 	LUA \
 	SHAPELIB ZZIP \
 	LIBNET TIME OS THREAD \
-	UTIL GEO MATH
+	UTIL GEO MATH JSON
```

Comment:
Ok, it may be too early to add this lib to the project, but later it need this, if the project want to access to the Reader.cpp code. Then I have to add it, otherwize I get a linker error...
On this position it is pointless, you are right